### PR TITLE
Patch release r1.3

### LINKED ISSUE #363: Local links within API documentation have to be release specific
**Problem description**

There is currently one link within quality-on-demand.yaml this is pointing to main instead to the tagged release tree (line 13, https://raw.githubusercontent.com/camaraproject/QualityOnDemand/main/documentation/API_documentation/resources/QoD_latency_overview.PNG). If the file will be updated for upcoming version, also the documentation of the released version is impacted.

**Expected behavior**

The link should go into the release tree, e.g. for r1.2 to https://raw.githubusercontent.com/camaraproject/QualityOnDemand/r1.2/documentation/API_documentation/resources/QoD_latency_overview.PNG

For the Fall24 / v0.11.0 that can be handled within a patch release (-> adding Fall24 label)

**Alternative solution**

Any proposal which will come from Release Management to handle https://github.com/camaraproject/ReleaseManagement/issues/101

**Additional context**

### LINKED ISSUE #381: Create release PR for r1.3 (patch release)
**Problem description**

Scope as listed in #368 is almost addressed.

**Expected action**

Prepare the release PR for the patch release.

**Additional context**

### PR DESCRIPTION
**Pull request by hdamker**
#### What type of PR is this?

* subproject management
* documentation

#### What this PR does / why we need it:

Update of README and changelog for patch release r1.3
Updating release numbers and API versions across all files
Replacing links to main branch with release specific links (closing the remaining issue from the patch release scope in #

#### Which issue(s) this PR fixes:

Fixes #381 
Fixes #363

#### Special notes for reviewers:

Create temporarily created branch `r1.3` from main to be able to test the links. That branch will need be deleted before the release tag can be created.

#### Changelog input

```
na

```

### CODE REVIEW

**Review by eric-murray — DISMISSED** [review]

**Review by RandyLevensalor — CHANGES_REQUESTED** [review]
@hdamker Can you please update documentation/API_documentation/qos-profiles-API-Readiness-Checklist.md to include that we now have a user story?

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,6 +16,89 @@
...
+This patch release contains the definition and documentation of
+* quality-on-demand v0.11.1
+* qos-profiles v0.11.1
+* qod-provisioning v0.1.1
+
+The API definition(s) are based on
+* Commonalities v0.4.0
+* Identity and Consent Management v0.2.1
+
+Note: this release is a patch release, containing some clarifications and updates within the documentation. The API versions are fully backward-compatible to the ones in r1.2. Please see there for the main changes compared to previews 0.10.x release.
```
Comment (jlurien) [inline CHANGELOG.md:31]:
```suggestion
Note: this release is a patch release, containing some clarifications and updates within the documentation. The API versions are fully backward-compatible to the ones in r1.2. Please see there for the main changes compared to previous 0.10.x release.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -15,6 +16,89 @@
...
+
+### Added
+* Added guidance for multi-SIM scenarios in `ìnfo.description` by @eric-murray in https://github.com/camaraproject/QualityOnDemand/pull/378
+
+### Changed
+* Authorization and authentication section in `info.description` updated with text from Identity & Consent Management text v0.2.1 by @hdamker in https://github.com/camaraproject/QualityOnDemand/pull/370
+* Updated the note for the retrieve POST calls by @RandyLevensalor in https://github.com/camaraproject/QualityOnDemand/pull/372
+
+### Fixed
+* Documented that the error response QUALITY_ON_DEMAND.DURATION_OUT_OF_RANGE can't be returned for /extend and will be removed within next version by @hdamker in https://github.com/camaraproject/QualityOnDemand/pull/382
```
Comment (jlurien) [inline CHANGELOG.md:50]:
```suggestion
* Documented that the error response QUALITY_ON_DEMAND.DURATION_OUT_OF_RANGE cannot be returned for /extend and will be removed within next version by @hdamker in https://github.com/camaraproject/QualityOnDemand/pull/382
```

File: README.md [hunk 3]
```diff
@@ -26,22 +26,22 @@ Repository to describe, develop, document and test the QualityOnDemand API famil
 * Note: Please be aware that the project will have frequent updates to the main branch. There are no compatibility guarantees associated with code in any branch, including main, until a new release is created. For example, changes may be reverted before a release is created. **For best results, use the latest public release**.
 
 
-* **The latest public release is [r1.2](https://github.com/camaraproject/QualityOnDemand/tree/r1.2) with the following API definitions:**
+* **The latest public release is [r1.3](https://github.com/camaraproject/QualityOnDemand/tree/r1.3) a patch release of r1.2 with the following API definitions:**
```
Comment (jlurien) [inline README.md:29]:
```suggestion
* **The latest public release is [r1.3](https://github.com/camaraproject/QualityOnDemand/tree/r1.3), a patch release of r1.2, with the following API definitions:**
```

**Review by jlurien — DISMISSED** [review]
LGTM with minor suggestions

**Comment by hdamker** [comment]
> @hdamker Can you please update documentation/API_documentation/qos-profiles-API-Readiness-Checklist.md to include that we now have a user story?

@RandyLevensalor Sure ... done in 2b223a8c0f83852cdb6edbdeb2e75b4cf77a4866

**Comment by hdamker** [comment]
> LGTM with minor suggestions

@jlurien Thanks for the review, all suggestions committed.

**Review by RandyLevensalor — APPROVED** [review]

**Review by eric-murray — APPROVED** [review]

**Review by jlurien — APPROVED** [review]

**Review by tanjadegroot — APPROVED** [review]

**Comment by tanjadegroot** [comment]
Hi Herbert, sorry for the delay. I reviewed and have some comments, but you may decide to apply them in the next release.

OK to go from release management team

**Comment by hdamker** [comment]
> Hi Herbert, sorry for the delay. I reviewed and have some comments, but you may decide to apply them in the next release.

Hi Tanja, thanks for the comments. I created a new issue to consider them within the next release, they are going beyond the scope of a release PR.

**Pull request merged**